components/typeform/actions/update-form/update-form.mjs

```diff
@@ -1,13 +1,14 @@
 import typeform from "../../typeform.app.mjs";
-import common from "../common.mjs";
+import utils from "../utils.mjs";
+
+const { reduceProperties } = utils;
 
 export default {
   key: "typeform-update-form",
   name: "Update a Form",
   description: "Updates an existing form. Request body must include all the existing form fields. [See the docs here](https://developer.typeform.com/create/reference/update-form-patch/)",
```

Comment:
description: "Updates an existing form's title. [See the docs here](https://developer.typeform.com/create/reference/update-form-patch/)",